test_runner/fixtures/utils.py

```diff
@@ -607,3 +624,19 @@ def human_bytes(amt: float) -> str:
         amt = amt / 1024
 
     raise RuntimeError("unreachable")
+
+
+def allpairs_versions():
+    """
+    Returns a dictionary with arguments for pytest parametrize
+    to test the compatibility with the previous version of Neon components
+    combinations were pre-computed to test all the pairs of the components with
+    the different versions.
+    """
+    ids = []
+    for pair in VERSIONS_COMBINATIONS:
+        cur_id = []
+        for state in sorted(pair.values()):
+            cur_id.append(state[0])
+        ids.append(f"combination_{''.join(cur_id)}")
```

Comment:
It seems ids is not set correctly, for example for `test_versions_mismatch[debug-pg17-combination` we have a bunch of the same `nnooo`:

<img width="633" alt="Screenshot 2024-10-10 at 14 34 56" src="https://github.com/user-attachments/assets/98647d0d-1a83-4304-84b5-0492e37ea888">

I guess `sorted` shouldn't be for values

Reply:
You are completely right.
Fixed, thanks!